### PR TITLE
Wizard Bandaid and Organization

### PR DESCRIPTION
**Pull request by Geevies**
While we wait for the wizard rework proposed by Kyres to be made, I decided to make a quick and dirty bandaid fix that, in my mind, would make the gamemode bearable until development finishes proper, to prevent pressure from rushing.

This bandaid features the removal of spatial wizard as a concept, as well as removing teleportation abilities from the other wizard classes. Instead, wizards get 12 SP instead of 8 SP (necromancers got 8 SP up from 5 SP) to choose spells that'll give them more survivability. Similarly, Wizard Apprentices get 8 SP instead of 3 SP, as well as a few more spells. This makes people more likely to pick apprentices, and allows the wizard to get more anti-sec survivability in the form of a bullet sponge.

* Removed Spatial as a possible Wizard Spellbook.
* Gave wizards and apprentices more spell points across the board in return for losing all teleportation abilities.
* Wizard apprentices gain mend, forcewall, shadow shroud, wizard voidsuit, monster manual, raise dead. Wizards should try to get apprentices where possible.

EDIT: I also decided to make the organization of the spell shit better.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Kaedwuff** [comment]
"It's really annoying when wizards always teleport away"
"Let's remove this iconic ability from Wizards entirely rather than addressing the greater problems of that they teleport away because security goes gung-ho with them and because we have created a medium where mobbing the antag is still a valid behavior and that wizards are not punished for making a build that is capable of contributing nothing but being annoying."
"Won't that mean a lot more wizards get killed or arrested because 'bullet sponge' isn't actually valid defense against being mobbed by security?"
"Yes, but I don't care apparently because my concept of balance involves removing entire features that frustrate people without considering their impact on the viability of the entire concept of a single antagonist with no real escape methods against an entire station."
"Are you going to add anything to compensate for this lack of ability to escape danger? Wizards other than necromancer liches don't really have the ability to tank, that was never the point of wizards in general."
"No, I'm just going to give people more spell points and point at kyres's vague promises of a future rework because I'm designating this a bandaid fix and that makes it okay."

**Comment by Geevies** [comment]
Okay. I bloated the hell out of this PR, as usual. However, the GAMEPLAY changes are in the FIRST commit. So if you want to review my buff and removal of spatial, take a look at the FIRST COMMIT. The second commit is the RESTRUCTURING of the spell folder, because it was ass. So, if you want to ensure I didn't sneak any changes in, look at the SECOND commit. (The only thing that was removed was the "summon horsemask" spell because it's an LRP leftover.) The THIRD commit is me fixing the .dme cuz i'm an idiot and forgot to do that in the second one. Have fun reviewing.

File: code/modules/spells/spellbook/student.dm [hunk 1]
```diff
@@ -16,12 +16,17 @@
...
-				/obj/item/gun/energy/staff/focus = 		1,
-				/obj/item/storage/belt/wands/full = 		2,
-				/obj/item/contract/wizard/xray = 		1
-					)
+	spells = list(/spell/targeted/mend =						1,
+				/spell/aoe_turf/knock =							1,
+				/spell/targeted/projectile/magic_missile =		1,
+				/spell/aoe_turf/conjure/forcewall =				1,
+				/spell/shadow_shroud =							1,
+				/spell/targeted/raise_dead =					2,
```
Comment (Doxxmedearly) [inline code/modules/spells/spellbook/student.dm]: Unsure about allowing them to raise dead.

Comment (Doxxmedearly) [inline code/modules/spells/spellbook/student.dm]:
```suggestion

```

File: code/modules/spell_system/spellbook/variants/student.dm [hunk 2]
```diff
@@ -16,12 +16,17 @@
...
-				/obj/item/gun/energy/staff/focus = 		1,
-				/obj/item/storage/belt/wands/full = 		2,
-				/obj/item/contract/wizard/xray = 		1
-					)
+	spells = list(/spell/targeted/mend =						1,
+				/spell/aoe_turf/knock =							1,
+				/spell/targeted/projectile/magic_missile =		1,
+				/spell/aoe_turf/conjure/forcewall =				1,
+				/spell/shadow_shroud =							1,
+				/spell/targeted/raise_dead =					2,
```
Comment (Geevies) [inline code/modules/spell_system/spellbook/variants/student.dm]:
```suggestion
```

**Comment by Geevies** [comment]
raise dead for students removed

**Pull request merged**